From 4f3d2dee87d40491ef4a2664e21168462f3bb681 Mon Sep 17 00:00:00 2001
From: Alexander Hendrich <hendrich@chromium.org>
Date: Sat, 27 Jun 2020 01:33:00 +0000
Subject: [PATCH 10/12] Disable flaky
 DiceManageAccountBrowserTest.ClearManagedProfileOnStartup [M83 merge]

TBR=msarda@chromium.org,hendrich@chromium.org

(cherry picked from commit 9eeb72ff8110366b2501321d4b7f6e3e470796e6)

Bug: 1075896
Change-Id: I6c5e9c60fffae5bb9fcbb7b29fb3ed1d6f71fbfd
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2170026
Reviewed-by: Alexander Hendrich <hendrich@chromium.org>
Commit-Queue: Alexander Hendrich <hendrich@chromium.org>
Cr-Original-Commit-Position: refs/heads/master@{#763258}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2268386
Reviewed-by: Carlos Knippschild <carlosk@chromium.org>
Commit-Queue: Carlos Knippschild <carlosk@chromium.org>
Cr-Commit-Position: refs/branch-heads/4103@{#733}
Cr-Branched-From: 8ad47e8d21f6866e4a37f47d83a860d41debf514-refs/heads/master@{#756066}
---
 chrome/browser/signin/dice_browsertest.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/signin/dice_browsertest.cc b/chrome/browser/signin/dice_browsertest.cc
index e0733ccb9eb..b362f578882 100644
--- a/chrome/browser/signin/dice_browsertest.cc
+++ b/chrome/browser/signin/dice_browsertest.cc
@@ -1015,8 +1015,9 @@ IN_PROC_BROWSER_TEST_F(DiceManageAccountBrowserTest,
       prefs::kSigninAllowedOnNextStartup, false);
 }
 
+// TODO(https://crbug.com/1075896) disabling test due to flakiness
 IN_PROC_BROWSER_TEST_F(DiceManageAccountBrowserTest,
-                       ClearManagedProfileOnStartup) {
+                       DISABLED_ClearManagedProfileOnStartup) {
   // Initial profile should have been deleted as sign-in and sign out were no
   // longer allowed.
   PrefService* local_state = g_browser_process->local_state();
-- 
2.28.0

